CHANGES.md

```diff
@@ -6,6 +6,10 @@ Version 5.1.3
 
 To be released.
 
+ -  Removed mutations from explorer.  [[#3898]]
```

Comment:
```suggestion
 -  (Libplanet.Explorer) Removed mutations from explorer.  [[#3898]]
```

I'm not sure this is enough to fix the issue. I think this should be tested on a node image first.

Reply:
https://github.com/planetarium/libplanet/pull/3898/commits/0e92909236c8f883b849b0179b3e9491d3f8f1f1 amended